README.rst

```diff
@@ -41,7 +41,22 @@ MNE-BIDS
 
 This is a repository for creating
 `BIDS <http://bids.neuroimaging.io/>`_\ -compatible datasets with
-`MNE <https://mne.tools/stable/index.html>`_.
+`MNE-Python <https://mne.tools/stable/index.html>`_.
```

Comment:
have you clarified the notion of what does it mean to be "BIDS compatible"? It means that it passes the validator or that it conforms to the standard. It may not be obvious to everyone